client/components/theme/style.scss

```diff
@@ -35,9 +37,14 @@ $theme-info-height: 54px;
 		}
 	}
 
-	&.is-actionable {
+	&:not( .is-actionable ) {
+		.theme__thumbnail {
+			cursor: default;
```

Comment:
This `cursor: default` property could be moved to the `.theme__thumbnail` rule, without a `:not` condition.

The `.theme.is-actionable .theme__thumbnail` selector will then always override it because of higher specificity.

Reply:
Yeah, I had it like that before -- I was mainly using the `:not()` selector since it seemed to add some clarity. But I'm fine with moving it out.